lib/tower_cli/models/base.py

```diff
@@ -312,9 +312,19 @@ def _format_yaml(self, payload):
                 """Convert the payload into a YAML string with proper
                 indentation and return it.
                 """
-                payload = json.loads(json.dumps(payload))
-                return yaml.safe_dump(payload, indent=2, allow_unicode=True,
-                                      default_flow_style=False)
+                def ordered_dump(data, stream=None, Dumper=yaml.Dumper, **kws):
```

Comment:
Thinking about this some more.... I wouldn't mind `ordered_dump` being a function kept in a workspace for functions. This might help out later with other items on the agenda. Perhaps `lib/utils/parsers.py`.